client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -98,7 +98,7 @@ export default connect( state => {
 
 	return {
 		showRewindCredentials:
-			rewind.state === 'awaitingCredentials' ||
+			rewind.state === 'awaiting_credentials' ||
```

Comment:
The root cause here may be the `[fromApi()](https://github.com/Automattic/wp-calypso/blob/fac6d4209d88acc1cbbd3b0f318b2d28aeecd080/client/state/data-layer/wpcom/sites/rewind/api-transformer.js#L59)` handler in the data layer not transforming to camel case for some reason.